### DISCUSSION TITLE
Supporto per Accordi

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/ods-mkdocs-material#53] [Domande]
Secondo voi (Material for) MkDocs è adatto per la pubblicazione di un canzoniere (con testo canti ed accordi)? Esiste un supporto per accordi che permetta di ottenere

```
Do    Lam   Rem       Sol
Lorem ipsum dolor sit amet
...
```
senza dover ricorrere alla formattazione per codice?

**Comment by aborruso** [comment]
Caro Dennis,
non ti so rispondere, non ho esperienza.
Facendo una ricerca mkdocs and chords, escono cose correlate, ma da capire.

Metto in elenco sparso:

- https://github.com/music-markdown/music-markdown 
- https://guidodoc.grame.fr/guide/chords/
- https://dnotes.github.io/markdown-it-chords/

Quest'ultimo mi sembra quello da esplorare, perché fa il rendering teoricamente in qualsiasi pagina HTML, includendo librerie. 
E quindi, modificando un po' il template di material, dovrebbe potersi integrare. Ma farei prime dei test HTML ciao mondo.

**Reply by dennisangemi** [reply]
Grazie @aborruso , preziosissimo come sempre. Avevo provato a cercare con scarsi risultati "mkdocs plugin chords" ma forse la carta vincente è utilizzare https://dnotes.github.io/markdown-it-chords/

Farò qualche test e vi aggiornerò qui. Grazie ancora!
